ui/partials/perf/comparechooserctrl.html

```diff
@@ -16,7 +20,11 @@
           <label for="new-project-selector">Project</label>
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
-          <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <div class="form-group has-error">
+            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <br/>
```

Comment:
ah, sorry about that! I always have code indent problem:(  I will fix this tomorrow. Can you tell me some tools can check out my code style like flask8 for Python? 

By the way, can we talk on irc about the other bug I work on? I still confuse about ng-class and I really want to know how to solve that. Thank you:)